src/test/java/uk/gov/cshr/report/integration/CourseCompletionsReportRequestIntegrationTest.java

```diff
@@ -72,6 +77,9 @@ public class CourseCompletionsReportRequestIntegrationTest extends IntegrationTe
 
     private static GenericContainer<?> azuriteContainer;
 
+    @MockBean
+    IUserAuthService userAuthService;
```

Comment:
We need to avoid mocking services in the integration tests. There's code to support adding roles/authorities to tokens in the course completions integration tests:

```
SecurityMockMvcRequestPostProcessors.JwtRequestPostProcessor identityDeletePostProcessor = jwt()
                .jwt(getJwt())
                .authorities(new SimpleGrantedAuthority("IDENTITY_DELETE"));